lib/tasks/delete.rake

```diff
@@ -39,4 +39,37 @@ namespace :delete do
     deleter = DuplicateDeleter.new(type_to_delete)
     deleter.call(links, id_type: 'link')
   end
+
+  # Can be removed once run in production
+  desc "
+  Delete old business support scheme items.
+  "
+  task :old_business_support do
+    require "indexer/workers/delete_worker"
+    elasticsearch_config = YAML.load("../../elasticsearch.yml")
```

Comment:
I discovered in another PR that this doesn't actually work - or rather, it only works because the Elasticsearch client falls back to `localhost:9200` if the URL is nil.

Fortunately you can use `SearchConfig.new.elasticsearch` instead. :)

https://github.com/alphagov/rummager/pull/811/commits/0210a1d2600d5bc15e4bc12c1119b6410cfe3b55